base_tier_validation/models/tier_review.py

```diff
@@ -150,15 +155,26 @@ def _compute_todo_by(self):
     def _get_reviewers(self):
         if self.reviewer_id or self.reviewer_group_id.users:
             return self.reviewer_id + self.reviewer_group_id.users
-        reviewer_field = self.env["res.users"]
         if self.reviewer_field_id:
-            resource = self.env[self.model].browse(self.res_id)
-            reviewer_field = getattr(resource, self.reviewer_field_id.name, False)
-            if not reviewer_field or not reviewer_field._name == "res.users":
-                raise ValidationError(
-                    self.env._("There are no res.users in the selected field")
+            return self._check_reviewers_field()
+        return self.env["res.users"]
+
+    def _check_reviewers_field(self):
+        res = False
+        rec = self.env[self.model].browse(self.res_id)
+        reviewer_field_val = getattr(rec, self.reviewer_field_id.name, False)
+        if reviewer_field_val:
+            if reviewer_field_val._name == "res.groups":
+                res = reviewer_field_val.users
+            if reviewer_field_val._name == "res.users":
+                res = reviewer_field_val
+        if not res:
+            raise ValidationError(
```

Comment:
Can you add a line in your test to check this error can be raised successfully? Should make codecov happy.